extension/manifest.json

```diff
@@ -9,34 +9,22 @@
           "background.js"], 
         "persistent": false
     },
-    "page_action": {
-        "default_popup": "popup.html",
+    "commands":{
+        "_execute_browser_action": {
+            "suggested_key": {
+                "default": "Alt+R"
```

Comment:
Added to meeting notes. https://workflowy.com/s/research-journal/qLzvlxUttujnlQ8c
Will submit this for now.

Reply:
Oh and this change will also affect how you want to implement "turn off"
Maybe now it will 1. call extension cleanup logic (remove highlight) and 2. inactivate extension.